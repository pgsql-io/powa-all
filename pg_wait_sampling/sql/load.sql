CREATE EXTENSION pg_wait_sampling;

\d pg_wait_sampling_current
\d pg_wait_sampling_history
\d pg_wait_sampling_profile

DROP EXTENSION pg_wait_sampling;
