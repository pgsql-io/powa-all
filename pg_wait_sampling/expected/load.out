CREATE EXTENSION pg_wait_sampling;
\d pg_wait_sampling_current
View "public.pg_wait_sampling_current"
   Column   |  Type   | Modifiers 
------------+---------+-----------
 pid        | integer | 
 event_type | text    | 
 event      | text    | 
 queryid    | bigint  | 

\d pg_wait_sampling_history
      View "public.pg_wait_sampling_history"
   Column   |           Type           | Modifiers 
------------+--------------------------+-----------
 pid        | integer                  | 
 ts         | timestamp with time zone | 
 event_type | text                     | 
 event      | text                     | 
 queryid    | bigint                   | 

\d pg_wait_sampling_profile
View "public.pg_wait_sampling_profile"
   Column   |  Type   | Modifiers 
------------+---------+-----------
 pid        | integer | 
 event_type | text    | 
 event      | text    | 
 queryid    | bigint  | 
 count      | bigint  | 

DROP EXTENSION pg_wait_sampling;
